### DISCUSSION TITLE
Bring your ideas to life with GitHub Spark [Technical Preview] ⚡

### DISCUSSION THREAD

**Discussion by ebndev** [community/community#142199] [Other Feature Feedback, Questions, & Ideas]
![unnamed](https://github.com/user-attachments/assets/ee69e0e2-6c21-4b50-8a96-3c2cee579045)

#### Sparks are flying Day 1 of GitHub Universe! 

It starts with an idea. It always does. When we’re kids, we have thousands of them. Many of them are silly. Some of them are crazy. Today, we are giving you a new product to try all of them. We call it GitHub Spark—it’s powered by natural language, and it sets the stage for our vision to help 1 billion people become developers. This is about fun and personal software, not about enterprise productivity apps. With live history, previews, and the ability to edit code directly, GitHub Spark allows you to create micro apps that take that crazy small, fun idea and bring it to life. Go and play… [Learn about the GitHub Spark technical preview >](https://gh.io/spark) ⚡

👋 If you’d like to try out GitHub Spark, then sign up for the technical preview [here](https://github.com/github_spark_waitlist_signup/join).

#### Take a look at how it works 🎬

- **Watch**: [What is GitHub Spark? Introducing a brand new way to build powerful, AI assisted applications](https://www.youtube.com/watch?v=oM2amcnVmzM)
- **Watch**: [Use GitHub Spark to create a travel log app](https://githubnext.com/projects/github-spark)
- Learn all about GitHub Spark & explore some sparks that the team made (and use!) on [GitHub Next](https://githubnext.com/projects/github-spark)
- New from Universe 2024: [Get the latest previews and releases](https://github.blog/news-insights/product-news/universe-2024-previews-releases/)

What do you wish you had this app for before you spent countless hours coding? Leave a comment and let us know what your first Spark will be!

**Comment by Serch193** [comment]
Buenas tardes disculpen me podrian ayudar a quitar kg de mi telefono samsung a03S  porfavor

**Comment by MasterInQuestion** [comment]
Well, but does it really solve the fundamental problems?..

    Rigorously defined description is the foundation for efficient operation:
    Trying to circumvent this complexity would only end in greater complexity or utter confusion.

**Reply by djredman99** [reply]
@MasterInQuestion Perhaps you are missing this: "This is about fun and personal software, not about enterprise productivity apps. "

**Reply by MasterInQuestion** [reply]
Maybe. But the very problems may as well ruin the fun...
    Something rooted too deep in the whole infrastructure, and not easy for a thorough fix.

**Comment by Ioana41** [comment]
FVL! Fiat vera lux!

**Comment by djesusnet** [comment]
O GitHub Spark tem um lado muito bacana, porque traz uma forma mais leve e divertida de criar. Ele permite que a gente transforme aquelas ideias que surgem do nada em pequenos aplicativos, e o melhor é que você não precisa ser um expert em programação para começar. Com a ajuda da IA e uma interface bem intuitiva, ele deixa tudo mais acessível e rápido, o que é ótimo para quem quer testar uma ideia ou simplesmente brincar um pouco com o código.

Por outro lado, como ainda está em fase inicial, pode ser que ele não tenha todas as funcionalidades ou a robustez que projetos maiores exigem. Talvez quem já está acostumado a trabalhar com algo mais complexo sinta falta de mais controle ou recursos. Mas, no geral, para quem quer explorar ideias, prototipar rapidamente, ou até se divertir desenvolvendo algo, o GitHub Spark tem um potencial gigante. É uma ferramenta que pode abrir portas para muita gente!

**Comment by kenichi-lover** [comment]
it's now a field of vast exploration,blending science,engineering,and artistry into tools that let humanity touch the stars.

**Comment by Ioana41** [comment]
May Peace reign on us!

**Comment by Fa9o0366** [comment]
Heeee Heeeee.......

در تاریخ شنبه ۹ نوامبر ۲۰۲۴،‏ ۶:۳۸ قبل‌ازظهر Ioana-Noemy Toma <
***@***.***> نوشت:

**Reply by Ioana41** [reply]
Merci! Mercy on us!

**Comment by Ioana41** [comment]
Merci! Mercy on us!

‪Le lun. 4 nov. 2024 à 10:37, ‫محمد بوصالح‬‎ ***@***.***> a
écrit :‬

**Comment by engaboud** [comment]
> ![غير مسمى](https://private-user-images.githubusercontent.com/164093511/381239935-ee69e0e2-6c21-4b50-8a96-3c2cee579045.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzExOTI0MDksIm5iZiI6MTczMTE5MjEwOSwicGF0aCI6Ii8xNjQwOTM1MTEvMzgxMjM5OTM1LWVlNjllMGUyLTZjMjEtNGI1MC04YTk2LTNjMmNlZTU3OTA0NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwOVQyMjQxNDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hN2ExYWQ3NTBhNzM5MTE0OGNmOWMxNWMwNTRmMWY5MjYwMGE0NDViMzhhZThlNGMyOTE5N2RkODllODlkNjAzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Hh0ehkEFn2TMYakEub-RpmwsfoijiUXGfLVCgoJwXcA)
> 
> #### شرارات تتطاير في اليوم الأول من GitHub Universe!
> يبدأ الأمر بفكرة. وهذا يحدث دائمًا. عندما كنا أطفالًا، كانت لدينا آلاف الأفكار. كثير منها سخيف. وبعضها مجنون. اليوم، نقدم لك منتجًا جديدًا لتجربته جميعًا. نطلق عليه اسم GitHub Spark — فهو يعمل بلغة طبيعية، ويضع الأساس لرؤيتنا لمساعدة مليار شخص على أن يصبحوا مطورين. يتعلق الأمر بالبرامج الشخصية الممتعة، وليس بتطبيقات الإنتاجية المؤسسية. بفضل التاريخ المباشر والمعاينات والقدرة على تحرير التعليمات البرمجية مباشرةً، يتيح لك GitHub Spark إنشاء تطبيقات صغيرة تأخذ تلك الفكرة الصغيرة المجنونة والممتعة وتجعلها حقيقة. انطلق واستمتع... [تعرف على المعاينة الفنية لـ GitHub Spark >](https://gh.io/spark) ⚡
> 
> 👋 إذا كنت ترغب في تجربة GitHub Spark، فقم بالتسجيل للحصول على المعاينة الفنية [هنا](https://github.com/github_spark_waitlist_signup/join) .
> 
> #### ألقي نظرة على كيفية عمله 🎬
> * **شاهد** : [ما هو GitHub Spark؟ تقديم طريقة جديدة تمامًا لبناء تطبيقات قوية بمساعدة الذكاء الاصطناعي](https://www.youtube.com/watch?v=oM2amcnVmzM)
> * **شاهد** : [استخدم GitHub Spark لإنشاء تطبيق سجل السفر](https://githubnext.com/projects/github-spark)
> * تعرف على كل شيء عن GitHub Spark واستكشف بعض الشرارات التي صنعها الفريق (واستخدمها!) على [GitHub Next](https://githubnext.com/projects/github-spark)
> * جديد من Universe 2024: [احصل على أحدث المعاينات والإصدارات](https://github.blog/news-insights/product-news/universe-2024-previews-releases/)
> 
> ما الذي كنت تتمنى لو كان لديك هذا التطبيق قبل أن تقضي ساعات لا حصر لها في البرمجة؟ اترك تعليقًا وأخبرنا بما سيكون عليه تطبيق Spark الأول الخاص بك!

**Reply by samus-aran** [reply]
Hi @engaboud, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be hiding any posts in languages other than English for now, including this one.
